11.curiosity.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #201.            a_vec = F.one_hot(action, num_classes = self.env.action_space.n).reshape(-1,self.env.action_space.n) # convert action from int to one-hot format
</pre><p>Line is too long. Suggestion (Black style):</p><pre class="ql-syntax" spellcheck="false"># convert action from int to one-hot format
a_vec = F.one_hot(
    action, num_classes=self.env.action_space.n
).reshape(-1, self.env.action_space.n) 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Curt-Park/rainbow-is-all-you-need/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>